src/components/profile/ProfilePage.jsx

```diff
@@ -1,10 +1,11 @@
 import React from "react";
 import GeneralSection from "./general/GeneralSection";
-
+import MainSection from "../profilePage/ProfilePage";
 const ProfilePage = () => {
   return (
     <>
       <GeneralSection />
+      <MainSection />
```

Comment:
mainsection needs a better name. this says "generalsection" "mainsection" which is confusingly two very similar things

`profilePage/ProfilePage` should just be `profile/ProfilePage`

Reply:
i believe this part isnt related to mine!